apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -101,7 +101,10 @@ var TopInstructions = React.createClass({
         </div>
         <div style={collapseStyle}>
           <div style={bodyStyle}>
-            <Instructions renderedMarkdown={procesMarkdown(this.props.markdown)}/>
+            <Instructions
+              renderedMarkdown={procesMarkdown(this.props.markdown)}
```

Comment:
Note that for this particular typo, I was at least consistent (the line at the beginning of the file is `var procesMarkdown = require('markdown');` so tests wouldn't have changed anything (there's no actual bug, just a typo).

The other typo is a legit bug :(